src/Magnum/Audio/Buffer.h

```diff
@@ -52,7 +55,13 @@ class Buffer {
             Mono8 = AL_FORMAT_MONO8,        /**< 8-bit unsigned mono */
             Mono16 = AL_FORMAT_MONO16,      /**< 16-bit signed mono */
             Stereo8 = AL_FORMAT_STEREO8,    /**< 8-bit interleaved unsigned stereo */
-            Stereo16 = AL_FORMAT_STEREO16   /**< 16-bit interleaved signed stereo */
+            Stereo16 = AL_FORMAT_STEREO16,   /**< 16-bit interleaved signed stereo */
+
+            MonoFloat32 = AL_FORMAT_MONO_FLOAT32, /**< @al_extension{AL_EXT,float32} */
+            StereoFloat32 = AL_FORMAT_STEREO_FLOAT32, /**< @al_extension{AL_EXT,float32}, 32-bit interleaved float */
+
+            MonoDouble = AL_FORMAT_MONO_DOUBLE_EXT, /**< @al_extension{AL_EXT,double}, 64-bit interleaved double */
+            StereoDouble = AL_FORMAT_STEREO_DOUBLE_EXT, /**< @al_extension{AL_EXT,double}, 64-bit interleaved double */
```

Comment:
I'll change it to use the command, though. I will need that in a few other places.